tests/performance-test/performance-test.sh

```diff
@@ -251,10 +251,11 @@ while true; do
     case $STAGE in
         "TARGET")
             TARGETS=$(curl --silent "http://$(oc get route prometheus -o jsonpath='{.spec.host}')/api/v1/targets")
-            QDR=$(echo "$TARGETS" | grep -o '"__meta_kubernetes_service_name":"qdr-white"')
+            QDRWHITE=$(echo "$TARGETS" | grep -o '"__meta_kubernetes_service_name":"qdr-white"')
+            # QDRTEST=$(echo "$TARGETS" | grep -o '"__meta_kubernetes_service_name":"qdr-test"')
```

Comment:
Is there some reason you're not looking for both here?

Reply:
If the new qdr-test has not been completely established yet, the test exits without any indication of what happened. I need to add a section in to wait on the qdr-test instantiation. I thought I would do it in a separate PR but I think I will go ahead and do it here